### PR TITLE
fix error handling for inactive currency

### PR DESCRIPTION
**Pull request by ahujsak**
fixes : https://app.asana.com/0/361770107085503/1124410096045732

### CODE REVIEW

File: src/swap/changelly.js [hunk 1]
```diff
@@ -94,18 +94,17 @@ async function getAddress (
     : addressInfo.publicAddress
 }
 
-function checkReply (reply: Object, request?: EdgeSwapRequest) {
```
Comment (swansontec) [inline src/swap/changelly.js]: I would have just made the `request` mandatory - it has these two parameters inside it, plus other things other error types might need in the future. If Changelly ever has region restrictions, you will need `request` for the `SwapPermissionError` constructor, for example.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # edge-exchange-plugins
 
+# 0.6.5 (2019-06-04)
```
Comment (swansontec) [inline CHANGELOG.md]: I generally like to see the release notes + version bump in a separate commit whose commit message *is* the version, like "v0.6.5".

File: src/swap/changelly.js (hunk 1 above)
Comment (ahujsak) [reply src/swap/changelly.js]: ok, I will make that change

**Comment by swansontec** [comment]
Thanks, looks good.

**Pull request merged**